Home page Community tab

samhita-alla opened the pull request:
Signed-off-by: Samhita Alla <aallasamhita@gmail.com>

https://github.com/flyteorg/flyte/issues/2176

@SandraGH5, I've added "Community Channels" to the footer but not the navbar because we already have a detailed view of all the community channels in the community tab, and having "Community Channels" seems repetitive. In the footer, I've removed all the social media links in lieu of the channels option.

samhita-alla committed:
Home page Community tab

Signed-off-by: Samhita Alla <aallasamhita@gmail.com>

SandraGH5 commented:
> Signed-off-by: Samhita Alla [aallasamhita@gmail.com](mailto:aallasamhita@gmail.com)
> 
> [flyteorg/flyte#2176](https://github.com/flyteorg/flyte/issues/2176)
> 
> @SandraGH5, I've added "Community Channels" to the footer but not the navbar because we already have a detailed view of all the community channels in the community tab, and having "Community Channels" seems repetitive. In the footer, I've removed all the social media links in lieu of the channels option.

Do you mean you will remove the "Stay Connected" section?

samhita-alla commented:
> > Signed-off-by: Samhita Alla [aallasamhita@gmail.com](mailto:aallasamhita@gmail.com)
> > [flyteorg/flyte#2176](https://github.com/flyteorg/flyte/issues/2176)
> > @SandraGH5, I've added "Community Channels" to the footer but not the navbar because we already have a detailed view of all the community channels in the community tab, and having "Community Channels" seems repetitive. In the footer, I've removed all the social media links in lieu of the channels option.
> 
> Do you mean you will remove the "Stay Connected" section?

@SandraGH5, no! You've asked me to add "Community Channels" to the community tab in the navbar, right? I'm saying adding it seems like repeating the social media channels. I've added it to the footer, though. 
I'm merging this PR now, let me know if the site looks ok to you.
